src/listener/opfer.js

```diff
@@ -0,0 +1,7 @@
+export default function (controller, repliedCallback = () => {}) {
+  const types = ['direct_message', 'direct_mention', 'mention'];
+  controller.hears(['opfer', 'opfa'], types, (bot, message) => {
+    bot.reply(message, 'Du Opfer!');
+    repliedCallback();
```

Comment:
`repliedCallback` is not necessary (it's only for asynchronous unit tests). Please remove it 👍